app/lib/applied_standard_deduction.rb

```diff
@@ -0,0 +1,43 @@
+class AppliedStandardDeduction
+  attr_accessor :tax_year
+
+  def initialize(tax_return:)
+    @tax_return = tax_return
+    @tax_year = tax_return.year
+    @filing_status = tax_return.filing_status
+    @intake = tax_return.intake
+  end
+
+  def applied_standard_deduction
+    return nil if @intake.is_ctc? && @intake.home_location_puerto_rico?
+    raise NotImplementedError, "Standard deduction rule not implemented for #{@tax_year}" unless StandardDeductions.base_deductions(tax_year: @tax_year).present?
+
+    StandardDeductions.base_deductions(tax_year: @tax_year)[@filing_status] + additional_blind_standard_deduction + additional_older_than_65_standard_deduction
+  end
+
+  private
+
+  def additional_older_than_65_standard_deduction
+    deductions = StandardDeductions.older_than_65_deductions(tax_year: @tax_year)
```

Comment:
this is probably so negligible, but since this only applies to people that are over 65, we could add a return at the top that gets out of here if neither primary or spouse is over the age limit.

Also, I noticed that the method on tax_return does a date comparison every time we call the method. its probably really negligible memory allocation wise, but i guess it does that calculation every time we call the method. if we really wanted to, we could call the method once, set it to a variable, and then use the same computed value for the comparison.

Or maybe I'm totally wrong and the method only does the computation once and there's some kind of caching, but i think thats only for active record queries that rails can do that.

```
primary_senior = @tax_return.primary_age_65_or_older?
spouse_senior = @tax_return.spouse_age_65_or_older?

return unless primary_senior or spouse_senior

case ...
when "single"
  return deductions[:single_filer] if primary_senior
...
```